Modules/_decimal/_decimal.c

```diff
@@ -1519,7 +1519,7 @@ init_extended_context(PyObject *v)
 #ifdef EXTRA_FUNCTIONALITY
 /* Factory function for creating IEEE interchange format contexts */
 static PyObject *
-ieee_context(PyObject *dummy UNUSED, PyObject *v)
+ieee_context(PyObject *self, PyObject *v)
```

Comment:
```suggestion
ieee_context(PyObject *Py_UNUSED(module), PyObject *v)
```
maybe this way?

Reply:
Don't you need the module to get it's state?

Reply:
Oh, indeed)

But looking on the other module level function (e.g. PyDec_GetCurrentContext), I should reject your suggestion for consistency.

Reply:
`ctxmanager_new` just uses `m` so it's not consistent, I would rather stick to module.